src/apps/sorting-visualizer/components/controller/controls.module.scss

```diff
@@ -100,3 +105,15 @@
     }
   }
 }
+
+@media screen and (width <= 767px) {
```

Comment:
We are following mobile first design.

`@media screen and (width > 767px)`

Also we have followed the width >= 960px in the project. Any reason for picking 767px